lib/matplotlib/transforms.py

```diff
@@ -115,7 +115,7 @@ def __getstate__(self):
     def __setstate__(self, data_dict):
         self.__dict__ = data_dict
         # turn the normal dictionary back into a WeakValueDictionary
-        self._parents = WeakValueDictionary(self._parents)
+        self._parents = WeakValueDictionary(self.__dict__['_parents'])
```

Comment:
I think you've found a bug with the setstate implementation, and I don't think it has previously been tested correctly.

How about simply doing ``data_dict['_parents']`` though. It may be more clear what is going on that way.

Would you also mind adding a test that ``self._parents`` *actually is* accessible after being restored?